### DISCUSSION TITLE
Question about tutorial: Create a Next.js App - Setup

### DISCUSSION THREAD

**Discussion by virenb** [vercel/next.js#16312] [Help] [answered]
Hello,

Starting the tutorial to get a little more familiar with Next.js. I followed the instructions and was just wondering if this page would need to amended by adding another answer open ('Read this page!') or change the files in this starter project?

URL: [https://nextjs.org/learn/basics/create-nextjs-app/setup](https://nextjs.org/learn/basics/create-nextjs-app/setup "Next.js learn page")

```
npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter"

cd nextjs-blog

npm run dev
```

Opened up the browser to localhost:3000 as instructed and am greeted with 'Read this page!'. 

![Screen Shot 2020-08-18 at 10 43 31 AM](https://user-images.githubusercontent.com/10731287/90528070-387b9080-e140-11ea-981f-c07b97b5dbe1.png)

The question in this part of the tutorial asks what you see on the page with the below two answers

- Welcome to Next.js!
- Hello Next.js!

![Screen Shot 2020-08-18 at 10 44 00 AM](https://user-images.githubusercontent.com/10731287/90528148-4c26f700-e140-11ea-9d02-8693c8dce9d8.png)

**Comment by lfades** [comment, marked as answer]
I followed your steps and the starter has `Welcome to Next.js!` as the title, can you provide more details here 🙏

**Reply by virenb** [reply]
Hi -- I retried it and now I'm also seeing the `Welcome to Next.js!` now. I'm not sure what happened yesterday, as I did the same set up and installation yesterday. Thanks for your response!
